config/co2eq_parameters.js

```diff
@@ -30,3 +54,16 @@ exports.renewableAccessor = (zoneKey, k, v) => {
           k === 'nuclear') ? 0 : 1;
   // TODO(bl): remove storage from renewable list?
 }
+
+exports.powerOriginRatio = function(zoneKey, fuelKey) {
+  //if no ratios found for that zoneKey, use defaults
+  const key = (Object.keys(co2eqParameters.fallbackZoneMixes[zoneKey].powerOriginRatios || {}).length === 0) ? zoneKey : 'defaults' ;
+  
+  return co2eqParameters.fallbackZoneMixes[key].powerOriginRatios[fuelKey]
+}
+
+exports.powerOriginRatioFuel = function(fuelKey) {
```

Comment:
It is used in our private repo. This is used to create a function with a predefined fuelKey so it can be passed to compute as a defaultExportCoefAccessor (see line 248 of coupledmodel.test.js)